tsl/test/expected/compression_insert-13.out

```diff
@@ -740,9 +740,9 @@ SELECT * FROM trigger_test ORDER BY 1 ,2, 5;
  Fri Dec 31 22:00:00 1999 PST |      1 |     0 |         |        
  Fri Dec 31 22:00:00 1999 PST |      1 |     0 |         |        
  Sat Jan 01 00:00:00 2000 PST |      1 |   111 | eleven  |     111
- Sat Jan 01 00:00:00 2000 PST |      1 |     1 |         |        
  Sat Jan 01 00:00:00 2000 PST |      1 |     0 |         |        
  Sat Jan 01 00:00:00 2000 PST |      1 |     0 |         |        
+ Sat Jan 01 00:00:00 2000 PST |      1 |     1 |         |        
```

Comment:
I think we should fix this test to produce deterministic sort order otherwise this might lead to flaky tests

Reply:
done